lib/validators/specValidator.ts

```diff
@@ -1,6 +1,9 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+/* tslint:disable-next-line:no-reference */
+/// <reference path="./../../types/sway.d.ts" />
+
```

Comment:
I presume the definite solution is to move these types to sway itself. Can you create an issue for this please ?

Reply:
https://github.com/Azure/oav/issues/262